### PR TITLE
installation: update links and minor wording changes 

### PR DESCRIPTION
**Pull request by kasper93**

### CODE REVIEW

File: source/community.html.haml [hunk 1]
```diff
@@ -21,9 +21,9 @@ title: Community
             Internet Relay Chat
 
           %p
-            This is the preferred way to discuss mpv use and development. Never ask to
-            ask - just state your question. Stick around until someone replies.
-
+            This is the preferred way to discuss mpv usage and development.
```
Comment (sfan5) [inline source/community.html.haml:24]: "use" is not incorrect and I think it sounds nicer

File: source/installation.html.haml [hunk 2]
```diff
@@ -19,24 +19,37 @@ title: Installation
             Windows
           %p
             All binary packages are unofficial third-party builds.
+            Except for those linked from the mpv-player/mpv repository,
+            each commit and pull request has associated test builds.
```
Comment (sfan5) [inline source/installation.html.haml:23]: I don't understand this sentence.

File: source/installation.html.haml [hunk 3]
```diff
@@ -19,24 +19,37 @@ title: Installation
...
                     :"cloud-download"
-      = package_row 'Scoop', 'https://github.com/lukesampson/scoop-extras/blob/master/bucket/mpv.json'
-      = package_row 'Scoop (git)', 'https://github.com/lukesampson/scoop-extras/blob/master/bucket/mpv-git.json'
+      = package_row 'Windows builds by zhongfly (git)',
+                    'https://github.com/zhongfly/mpv-winbuild/releases',
+                    :"cloud-download"
+      = package_row 'First-party test builds on GitHub Actions',
+                    'https://github.com/mpv-player/mpv/actions/workflows/build.yml?query=event:push+branch:master',
+                    :"cloud-download"
+      = package_row 'Scoop', 'https://github.com/ScoopInstaller/Extras/blob/master/bucket/mpv.json'
```
Comment (sfan5) [inline source/installation.html.haml:34]: we could link this instead: https://scoop.sh/#/apps?q=mpv&id=b05b47128464d8969416289383fbfc69a47353e3

File: source/installation.html.haml (hunk 2 above)
Comment (kasper93) [reply source/installation.html.haml:23]: I'm trying to say that all binary build are third-party, except those that are in our (mpv-player/mpv) CI artifacts

File: source/community.html.haml (hunk 1 above)
Comment (kasper93) [reply source/community.html.haml:24]: restored

File: source/installation.html.haml (hunk 2 above)
Comment (kasper93) [reply source/installation.html.haml]: Changed it a little, maybe it's better.

File: source/installation.html.haml (hunk 3 above)
Comment (kasper93) [reply source/installation.html.haml]: changed and also did the same for MSYS2 link

**Review by sfan5 — APPROVED** [review]

**Pull request merged**